Source/UltraleapTrackingCore/Private/LeapDeviceWrapper.h

```diff
@@ -80,6 +80,12 @@ class FLeapDeviceWrapper : public FLeapWrapperBase
 		return Device.Get();
 	}
 
+	/** Used to set device hints
+	 * @param Hints - The device hints
+	 * @param LeapDeviceID - Device ID to set the hints
+	 */
+	virtual void SetDeviceHints(TArray<FString>& Hints, const uint32_t LeapDeviceID = 0) override;
```

Comment:
Do we reference the list of possible hints anywhere in the API - e.g. by pointing to the documentation, That would be a useful change.

Reply:
This is not available yet publicly https://ultrahaptics.slack.com/archives/C029XFH4085/p1710866158996979